Makefile

```diff
@@ -11,20 +11,24 @@ DEPLOY_ENV_VALID_CHARS=$(shell if echo $(DEPLOY_ENV) | grep -q '^[a-zA-Z0-9-]*$$
 LOGSEARCH_BOSHRELEASE_TAG=v209.0.0
 LOGSEARCH_FOR_CLOUDFOUNDRY_TAG=v207.0.0
 
+.PHONY:	check-env
```

Comment:
Extra space

Reply:
This was a tab, fixed in most recent commit - good spot